### PR TITLE
podman-run: clean up some formatting issues

### PR DESCRIPTION
**Pull request by miabbott**
I was browsing this page from the repo and some of the formatting was ugly.

This is a shot at cleaning some of it up.

Rendered version can be seen here:  https://github.com/miabbott/libpod/blob/fmt-podman-run/docs/podman-run.1.md

### CODE REVIEW

**Comment by rhatdan** [comment]
@miabbott Sign your PR, please.

**Comment by rhatdan** [comment]
@miabbott Did you verify these still look good as man pages as well as when viewed in markdown?

**Comment by miabbott** [comment]
@rhatdan I did not....so this PR is probably going to need more work

**Comment by miabbott** [comment]
@rhatdan Looks like I got lucky with my changes...man page looks reasonable, nothing broken AFAICT.

I force pushed a new commit that I've signed.

**Comment by rhatdan** [comment]
LGTM @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

File: docs/podman-run.1.md [hunk 1]
```diff
@@ -790,11 +812,7 @@ You would have to write policy defining a `svirt_apache_t` type.
 If you want to set `/dev/sda` device weight to `200`, you can specify the device
 weight by `--blkio-weight-device` flag. Use the following command:
 
-   # podman run -it --blkio-weight-device "/dev/sda:200" ubuntu
-
-```
-$ podman run -d busybox top
```
Comment (wking) [inline docs/podman-run.1.md]: Was this example intentionally dropped?  It's originally from 8cf07b2ad1 (#2) in case that helps.

Comment (miabbott) [reply docs/podman-run.1.md]: It didn't look like it fit in with the `--blkio-weight-device` example.

I could be misunderstanding how that example works, though.

Comment (TomSweeneyRedHat) [reply docs/podman-run.1.md]: @wking I was OK removing it as I didn't see how it applied to the vebiage above is at line 793 does.  @mheon @baude any thoughts?

**Pull request closed without merging**